alts/worker/runners/base.py

```diff
@@ -162,6 +165,46 @@ def repositories(self):
     def env_name(self):
         return self._env_name
 
+    def __init_task_logger(self, log_file):
+        """
+        Task logger initialization, configures a test task   logger to write
+        output to the given log file.
```

Comment:
I'd say it's not about logger initialization, it's more about adding a new handler for logger